web/themes/custom/cloudy/package.json

```diff
@@ -12,10 +12,16 @@
     "clean:pl": "rm -rf ../../../pattern-lab",
     "clean:wp": "rm -rf dist",
     "clean:migration1557": "rm -rf css js dist-cloudy dist-pl source",
+    "lint": "npm-run-all --parallel --continue-on-error --print-label lint:js lint:css",
+    "lint:css": "stylelint ./**/*.scss",
+    "lint:css:fix": "stylelint ./**/*.scss --fix",
+    "lint:fix": "npm-run-all --sequential --continue-on-error --print-label lint:js:fix lint:css:fix",
+    "lint:js": "eslint . --config ./.eslintrc.json",
+    "lint:js:fix": "eslint . --config ./.eslintrc.json --fix",
```

Comment:
The use of the paths in `eslint .` and `stylelint **/*.scss` will result in slower commands due to them going into `node_modules` to build the list even though they then exclude them afterwards. Better to point them to sub directories first.